modules/ROOT/pages/database-administration/aliases/remote-database-alias-configuration.adoc

```diff
@@ -132,92 +183,246 @@ dbms.security.key.name=key-name
+----
++
+[NOTE]
+====
+If a transaction modifies an alias (e.g. changing the database targeted on *DBMS B*), other transactions concurrently executing against that alias may be aborted and rolled back for safety.
+This prevents issues such as a transaction executing against multiple target databases for the same alias.
+====
+
+==== Set up SSO on the local DBMS and map the identity provider groups to the Neo4j roles
+
+In order for _Carol_ to get access to the remote database alias, she needs to be in an identity provider group that is mapped to a Neo4j role that is granted access to that alias.
+
+You set up SSO on the local *DBMS A* and map the identity provider groups to the Neo4j roles.
+For details, see the xref:tutorial/tutorial-sso-configuration.adoc[SSO configuration tutorial] and xref:authentication-authorization/sso-integration.adoc#auth-sso-map-idp-roles[Map the identity provider groups to the Neo4j roles].
+
+[parameters]
+----
+dbms.security.oidc.<provider>.well_known_discovery_uri=http://example.com/.well-known/discovery
+<...>
+dbms.security.oidc.<provider>.claims.groups=groups
+dbms.security.oidc.<provider>.authorization.group_to_role_mapping= "engineers" = admin; \
+                                                                   "collaborators" = reader; \
+                                                                   "remote_users" = remote_access
+----
+
+=== Configure the remote DBMS B (_Bob_)
+
+As _Bob_, you are responsible for setting up the remote *DBMS B*.
+You can create and delete users and grant or deny privileges on the databases managed by *DBMS B*.
+
+In this example, you need to ensure that _Carol_ can access `db1` on *DBMS B* using OIDC credential forwarding.
+
+. Create a custom role or chose an existing one, the roles will be assigned to the users on log in via the mapping of identity provider groups to the Neo4j roles:
++
+[source, Cypher]
+----
+CREATE ROLE db1_access
+----
+. Grant the `db1_access` role access to `db1`.
++
+[source, Cypher]
+----
+GRANT ACCESS ON DATABASE db1 TO db1_access
+GRANT MATCH {*} ON GRAPH db1 TO db1_access
+----
+
+. Set up SSO on the remote *DBMS B* and map the identity provider groups to the Neo4j roles.
+For details, see the xref:tutorial/tutorial-sso-configuration.adoc[SSO configuration tutorial] and xref:authentication-authorization/sso-integration.adoc#auth-sso-map-idp-roles[Map the identity provider groups to the Neo4j roles].
+//If you do not want specific users to access `db2`, here is where you set it.
++
+[parameters]
+----
+dbms.security.oidc.<provider>.well_known_discovery_uri=http://example.com/.well-known/discovery
+<...>
+dbms.security.oidc.<provider>.claims.groups=groups
+dbms.security.oidc.<provider>.authorization.group_to_role_mapping= "engineers" = admin; \
+                                                                   "collaborators" = reader; \
+                                                                   "remote_users" = db1_access
```

Comment:
Here, the role is different from the one configured on DBMS A. Don't they need to be the same?